DigitalLearningSolutions.Web/Views/LearningPortal/SelfAssessments/_SearchSelfAssessmentOverview.cshtml

```diff
@@ -18,10 +21,14 @@
   </div>
   <div class="filter-value-container">
     @Html.DropDownListFor(m => m.ResponseStatus,
-      Model.Filters.First().FilterOptions.Select(f => new SelectListItem(
-        text: f.DisplayText,
-        value: f.FilterValue.ToString(),
-        selected: f.FilterValue == SelfAssessmentCompetencyFilter.NotYetResponded.ToString())
+      Model.Filters.First().FilterOptions
+        .Where(f => parent.SelfAssessment.IsSupervised == false

```

Comment:
"== false" needs removing from this. Currently, it is limiting the selection when the self assessment is supervised but showing them all when it is not (the opposite of what is required). 
Looking at the implementation, we should actually be checking for `parent.selfAssessment.IsSupervisorResultsReviewed` rather than `parent.SelfAssessment.IsSupervised` too.